uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -410,19 +411,46 @@ Arguments:
 sub PatientNameMatch {
     my $this         = shift;
     my ($dicom_file) = @_;
-    my $cmd          = "dcmdump $dicom_file | grep PatientName";
+
+    my $lookupCenterNameUsing = NeuroDB::DBI::getConfigSetting(
+        $this->{'dbhr'},'lookupCenterNameUsing'
+    );
+
+    unless ( $lookupCenterNameUsing ) {
+        my $message = "\nConfig Setting 'lookupCenterNameUsing' is not set in "
+                      . "the Config module under the Imaging Pipeline section.";
+        $this->spool($message, 'Y', $notify_notsummary);
+        #TODO: once refactoring of exit code merged to minor, add the following
+        #TODO: MISSING_CONFIG_SETTING exit code in the generic database code
+        exit $NeuroDB::ExitCodes::MISSING_CONFIG_SETTING;
+    }
+
+    unless ($lookupCenterNameUsing =~ /PatientName|PatientID/i) {
```

Comment:
Since we expected either 'PatientName' or 'PatientID', shouldn't the pattern check be `/^(PatientName|PatientID)$/i`?